tests/network/wireguard.pm

```diff
@@ -75,7 +75,6 @@ sub run {
         assert_script_run 'modprobe wireguard';
     }
 
```

Comment:
How about instead of completely removing this test, just skip the check if the file doesn't exists?

```suggestion
assert_script_run('grep -i CONFIG_WIREGUARD /boot/config-$(uname -r)') unless(script_run('stat /boot/config-$(uname -r)') != 0);
```